### ISSUE TITLE
zip warning: name not matched: ../../bin/terraform-provider-ignition during 

### ISSUE THREAD

**Issue by todd-dsm** [openshift/installer#5833]
Not terribly clear on what's happening here but there is a build issue that needs to be resolved. This is the first step in the README file.

# Version
NA

# Platform:
NA, this is about `hack/build.sh`

Please specify:
I don't know, so it must be IPI

# What happened?

The README says...

> Clone this repository. Then build the `openshift-install` binary with:

```sh
hack/build.sh
...
# golang.org/x/sys/unix
vendor/golang.org/x/sys/unix/syscall_darwin.1_13.go:29:3: //go:linkname must refer to declared function or variable
vendor/golang.org/x/sys/unix/zsyscall_darwin_amd64.1_13.go:27:3: //go:linkname must refer to declared function or variable
vendor/golang.org/x/sys/unix/zsyscall_darwin_amd64.1_13.go:40:3: //go:linkname must refer to declared function or variable
vendor/golang.org/x/sys/unix/zsyscall_darwin_amd64.go:28:3: //go:linkname must refer to declared function or variable
vendor/golang.org/x/sys/unix/zsyscall_darwin_amd64.go:43:3: //go:linkname must refer to declared function or variable
vendor/golang.org/x/sys/unix/zsyscall_darwin_amd64.go:59:3: //go:linkname must refer to declared function or variable
vendor/golang.org/x/sys/unix/zsyscall_darwin_amd64.go:75:3: //go:linkname must refer to declared function or variable
vendor/golang.org/x/sys/unix/zsyscall_darwin_amd64.go:90:3: //go:linkname must refer to declared function or variable
vendor/golang.org/x/sys/unix/zsyscall_darwin_amd64.go:105:3: //go:linkname must refer to declared function or variable
vendor/golang.org/x/sys/unix/zsyscall_darwin_amd64.go:121:3: //go:linkname must refer to declared function or variable
vendor/golang.org/x/sys/unix/zsyscall_darwin_amd64.go:121:3: too many errors
	zip warning: name not matched: ../../bin/terraform-provider-ignition

zip error: Nothing to do! (../../bin/terraform-provider-ignition.zip)
make: *** [Makefile:25: bin/terraform-provider-ignition.zip] Error 12
make: Leaving directory '/Users/tthomas/code/ibm/dnb/os/installer/terraform'
```

There is no `.openshift_install.log` at this point.

# What you expected to happen?

The expectation is this process should output a binary to the target location; 
The README says: `bin/openshift-install`
The `hack/build.sh` suggests: `cmd/openshift-install` (last line of the script)

Reproduce the issue

1. Begin with a macOS
2. Install GNU Tools: sed, make, grep, etc
3. attempt to run the installer: `hack/build.sh`

**Comment by patrickdillon** [comment]
Looks like the terraform-provider-ignition is failing due to https://github.com/golang/go/issues/49219 

The upstream provider needs to be updated or the provider removed from the installer.

**Comment by sosiouxme** [comment]
This will block our move to go 1.18 in 4.11 which is needed before feature freeze.

**Comment by patrickdillon** [comment]
Fixed by #5896 and #5865

/close

**Comment by patrickdillon** [comment]
@todd-dsm Thanks for submitting this issue btw. This was very helpful in identifying this problem early.
